Source/ChocolateyGui.Common.Windows/Services/PackageArgumentsService.cs

```diff
@@ -0,0 +1,64 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright company="Chocolatey" file="PackageArgumentsService.cs">
+//   Copyright 2017 - Present Chocolatey Software, LLC
+//   Copyright 2014 - 2017 Rob Reynolds, the maintainers of Chocolatey, and RealDimensions Software, LLC
+// </copyright>
+// --
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using chocolatey;
+
+namespace ChocolateyGui.Common.Windows.Services
+{
+    public class PackageArgumentsService : IPackageArgumentsService
+    {
+        private static readonly byte[] EntropyBytes = Encoding.UTF8.GetBytes("Chocolatey");
+
+        public string Decrypt(string id, string version)
+        {
+            var packageFolder = "{0}.{1}";
+            var packageId = id;
+            var packageVersion = version;
+            var chocolateyPath = @"C:\ProgramData\chocolatey\.chocolatey";

```

Comment:
As we discussed, we need to not hard code this. We already have a ChocolateyConfigurationProvider which has the information that we want, in the `ChocolatyInstall` property, and from there, we would just need to combine with the `.chocolatey` folder: https://github.com/chocolatey/ChocolateyGUI/blob/develop/Source/ChocolateyGui.Common/Providers/ChocolateyConfigurationProvider.cs